tools/gradle/README.md

```diff
@@ -0,0 +1,193 @@
+# Ice Slice Tools Gradle Plugin
```

Comment:
I think it would be clearer to have a tools subdir per language, like:

- java/tools or java/tools/gradle
- csharp/tools/ZeroC.Ice.Slice.Tools.Csharp
- cpp/tools/ZeroC.Ice.Slice.Tools.Cpp

with the common ZeroC.Ice.Slice.Tools probably in csharp/tools.

Reply:
Fine with me. I started with the top-level tools because that is what we did in IceRPC. It was also convenient to share code among the C++ and C# projects.